docs/basic-features/typescript.md

```diff
@@ -170,3 +160,23 @@ module.exports = nextConfig
 Since `v10.2.1` Next.js supports [incremental type checking](https://www.typescriptlang.org/tsconfig#incremental) when enabled in your `tsconfig.json`, this can help speed up type checking in larger applications.
 
 It is highly recommended to be on at least `v4.3.2` of TypeScript to experience the [best performance](https://devblogs.microsoft.com/typescript/announcing-typescript-4-3/#lazier-incremental) when leveraging this feature.
+
+## Ignoring TypeScript Errors
+
+Next.js fails your **production build** (`next build`) when TypeScript errors are present in your project.
+
+If you'd like Next.js to dangerously produce production code even when your application has errors, you can disable the built-in type checking step.
+
+> Be sure you are running type checks as part of your build or deploy process, otherwise this can be very dangerous.
```

Comment:
```suggestion
Be sure you are running type checks as part of your build or deploy process, otherwise this can be very dangerous.
```
Should this be in a quote?